extractor/src/extractors/BuildGradleExtractor.ts

```diff
@@ -55,6 +55,7 @@ export default class BuildGradleExtractor implements Extractor {
                     module: parentModule,
                     versionConstraint: parentVersion,
                     scopes: [ "parent" ],
+                    name: [ parentOrganization, parentModule ].join(":"),
```

Comment:
+1. I wasn't specific about this, but this is the joining semantic I was expecting.

Reply:
Updated!